tests/test_docs.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+import warnings
+
+from conftest import get_json_files, get_examples_dir, file_id
+from jscc.testing.checks import get_empty_files, get_misindented_files, get_invalid_json_files
+
+
+"""
+Docs tests
+
+Checks the example data used in the documentation are all valid BODS.
+"""
+
+
+examples = get_json_files(get_examples_dir())
+
+
+def test_examples_empty(examples_dir):
+    """
+    Check the data in the /examples directory are not empty
+    """
+    errors = False
+    empties = get_empty_files(top=examples_dir)
+    for path in empties:
+        warnings.warn(f'ERROR: Empty file at {path}')
+        errors = True
+
+    assert not errors, 'Empty files found, see warnings.'
+
+
+def test_examples_indent(examples_dir):
+    """
+    Check the data in the /examples directory are indented correctly.
+    """
+    errors = False
+    misindented = get_misindented_files(top=examples_dir)
+    for path in misindented:
+        warnings.warn(f'ERROR: Misintended files at {path}')
```

Comment:
typo - misintended should be misindented

Reply:
I have fixed the typo.
https://github.com/openownership/data-standard/pull/565